tests/e2e/convolution/BUILD.bazel

```diff
@@ -0,0 +1,57 @@
+# Copyright 2022 The IREE Authors
+#
+# Licensed under the Apache License v2.0 with LLVM Exceptions.
+# See https://llvm.org/LICENSE.txt for license information.
+# SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+# End-to-end convolution 2d tests.
```

Comment:
Any particular reason to limit this to conv2d? What about conv1d and conv3d?

Reply:
There are more variables involved. In fact, conv2d has a larger search space than matmul 😄. I can add that in the future once we have the base set.